packages/rest/src/__tests__/acceptance/coercion/coercion.acceptance.ts

```diff
@@ -98,9 +115,9 @@ describe('Coercion', () => {
       // This is because all values are encoded as strings on URL queries
       // and we did not specify any schema in @param.query.object() decorator.
       where: {
-        id: '1',
```

Comment:
Do we want to add `fields` filter to this test case? You might want to remove the comment above as it is fixed.

Reply:
@agnes512 I started adding feature by manually verify the field filter in todo app, so the fields filter does work.

`where: {active: true}`
and 
`fields: {name: true}`
doesn't make any difference for testing purpose, so I would say the current test case is good enough to cover our case(field filter) :p

Reply:
Oh I mentioned it because I remember when I was testing it, it works for `where` for some reasons 😂

Reply:
@agnes512 I removed my change and tested with todo app, filter `?[where][isComplete]=false` does return correct result for some reason, weird. 

But the filter it self is not coerced, see my screenshot in:
<img width="625" alt="Screen Shot 2020-06-23 at 4 13 56 PM" src="https://user-images.githubusercontent.com/12554153/85456282-c9812180-b56c-11ea-9052-17d9eb6b2e7c.png">

isComplete is `"false"` not `false`.

I need more time to investigate the reason, but the updated test case here should be correct. For the parameter coerce function, it wouldn't make sense if

`where: {active: true}` get coerced while `fields: {name: true}` not.